Modules/_datetimemodule.c

```diff
@@ -3325,7 +3325,7 @@ static PyTypeObject PyDateTime_IsoCalendarDateType = {
     .tp_doc = iso_calendar_date__doc__,
     .tp_methods = iso_calendar_date_methods,
     .tp_getset = iso_calendar_date_getset,
-    .tp_base = &PyTuple_Type,
+    // .tp_base = &PyTuple_Type,  // filled in PyInit__datetime
```

Comment:
The `xxsubtype.c` module does this weird thing with a macro called `DEFERRED_ADDRESS` that always returns `0`: https://github.com/python/cpython/blob/60398512c86c5535edd817c99ccb50453b3b0471/Modules/xxsubtype.c#L134

Not sure exactly why it's done that way, though.

Reply:
It's just documentation: https://github.com/python/cpython/blob/60398512c86c5535edd817c99ccb50453b3b0471/Modules/xxsubtype.c#L10-L17
Would you like to use `DEFERRED_ADDRESS` instead?